app/assets/javascripts/app/views/hovercard_view.js

```diff
@@ -94,14 +95,20 @@ app.views.Hovercard = app.views.Base.extend({
 
   _populateHovercard: function() {
     var href = this.href();
-    href += "/hovercard.json";
+    href += ".json";
```

Comment:
That's true, I wanted opinions on this topic. The main point of this change was to remove HovercardPresenter. I can introduce hovercard presenting method for PersonPresenter that strips bio and other fields and bring the `/hovercard.json` route back.